<launch>
  <node pkg = "ransac_project" type = "nodeVisual" name = "nodeVisual" />
</launch>
